common/src/main/proto/TransportMessages.proto

```diff
@@ -690,6 +705,8 @@ message PbOpenStream {
   int32 endIndex = 4;
   int32 initialCredit = 5;
   bool readLocalShuffle = 6;
+  bool requireSubpartitionId = 7;
+  bool shuffleDataNeedSort = 8;
```

Comment:
non-essential modification

Reply:
thanks, rebase code occurs some mistakes, will be updated later